dev/how-to/upgrade/tidb-in-kubernetes.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Rolling Update TiDB in Kubernetes
+category: how-to
```

Comment:
Please refer to other documents and also add a `summary` line (using a complete sentence) after the `title` line.